doc/api/cli.md

```diff
@@ -298,6 +298,17 @@ added:
 
 Expose the [Web Crypto API][] on the global scope.
 
+### `--experimental-import=module`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+Preload the specified module at startup.
+
+Follows [ECMAScript modules][] resolution rules.
+Use \[`--require`]\[] to load a \[CJS module]\[].
```

Comment:
We don't want to escape the `[` here, instead we want to create the ad-hoc references at the end of the file.
```suggestion
Use [`--require`][] to load a [CJS module][].
```